### ISSUE TITLE
Super Admin your admin

### ISSUE THREAD

**Issue by poeticjustice** [SocialEngine/phpv4-feature-requests#123]
Forgive in advance incase I have completely missed something and this can be achieved already.

So I am admin of my site, and I want other admins on the site but with less acess but still have access to admin area.

I dont want them seeing every users email addresses, I dont want them seeing the plugins, i dont want them to have access to theme options or layout, I dont want them seing the licence number etc, 

Can this be done?

**Comment by gsf00001** [comment]
Yes, this is a great idea.  I asked about this (actually, the ability to control access throughout ADMINcp) at the community site, and I thought I created an issue here or added to someone elses (but I just searched and can't find anything).  I don't want to hijack this thread, so my only suggestion is to go beyond what you're mentioning and do this throughtout ADMINcp (including for 3rd-Party Plugins) since there are many things that should be controlled/limited by different ADMIN levels.

**Comment by poeticjustice** [comment]
exactly that @gsf00001 there is certain admin i need, but i dont want them having full AdminCP Access

**Comment by DonnaScriptTechs** [comment]
Just a quick note that I removed the license details on my sites as I write tutorials and don't want them seeing that but I thought I saw someplace that we would be removing it from showing for upcoming improvements. I'll have to go look.

**Comment by poeticjustice** [comment]
Id imagine you could custom css and set display none, but i think it would be great to hide it

**Comment by DonnaScriptTechs** [comment]
I didn't do that. I completely removed it as there's no reason to have it IMO.

**Comment by poeticjustice** [comment]
completely removed it ??

**Comment by DonnaScriptTechs** [comment]
Yes from the code I removed the display of the license,  not the licensing itself, just the display of it.

**Comment by poeticjustice** [comment]
May do that myself @DonnaScriptTechs but i dont know where it is in the code to do that so i would have to set to display none.

But this has kinda sidetracked, Is there a way to limit what admin sees, if super admin doesnt want an admin see everything in the backend

**Comment by DonnaScriptTechs** [comment]
Right now, I don't think so other than the settings that exist for some things. I know how to hide emails in code and the license but hadn't seen a setting to hide that and plugins.

**Comment by poeticjustice** [comment]
Be interesting toknow how you did this @DonnaScriptTechs if your ever on the community site :)
